basic/datadog_log_destination/datadog_log_destination.tf

```diff
@@ -0,0 +1,53 @@
+############ Creating a Kinesis Firehose Delivery Stream to Datadog ##############
+
+# This module creates a AWS Kinesis Firehose Delivery Stream that pipes logs to Datadog
+module "datadog_connector" {
+  source  = "symopsio/datadog-connector/aws"
+  version = ">= 2.0.0"
+
+  environment = "main"
+
+  # This variable should NOT be checked into version control!
+  # Set it in an untracked tfvars file (e.g. `secrets.tfvars`)
+  # or as an environment variable: `export TF_VAR_datadog_access_key="my-access-key"`
+  datadog_access_key = var.datadog_access_key
+}
+
+resource "sym_log_destination" "datadog" {
+  type = "kinesis_firehose"
+
+  # The Runtime Permission Context has Kinesis Firehose permissions from the aws/kinesis-firehose add-on
```

Comment:
There is no more add-on since we're not using the runtime-connector module anymore.